develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/02-liferay-util-body-top.markdown

```diff
@@ -0,0 +1,27 @@
+# Using Liferay Util Body Top [](id=using-liferay-util-body-top)
+
+The body top tag is not a self-closing tag. The content placed between the 
+opening and closing of this tag is moved to the top of the `body` tag. When 
+something is passed using this taglib, the 
+[body_top.jsp](https://github.com/liferay/liferay-portal/blob/7.1.x/portal-web/docroot/html/common/themes/body_top.jsp#L25-L31) 
+is passed markup and outputs in this JSP. 
+
+The example configuration below uses the `<liferay-util:body-top>` tag to 
+include JavaScript provided by the portlet's bundle:
+
+    <liferay-util:body-top outputKey="contenttop" >
```

Comment:
Shouldn't `outputKey` should be `bodytop` for consistency with the one on top?

Reply:
Yep. That makes sense. Thanks!